docs/Meadow/Meadow.OS/Networking/index.md

```diff
@@ -83,10 +83,15 @@ If you're configuring to join the network this way, you'll also need to edit the
 Credentials:
   Ssid: YourSSID
   Password: SSIDPassword
+# ClearDefaultCredentials: true
 ```
 
 In this case, you might want to register the `NetworkConnected` event in the `Initialize()` method to start any network related tasks once the device joins your network.
 
+Optionally you can set the `ClearDefultCredentials: true` , to remove the last credentials stored. Once you have done that you can comment on this property or set it to `false` and add new credentials.
```

Comment:
I dont fully undestand this feature. If I set my credentials and leave `ClearDefultCredentials: true`, will I still join the network?

Reply:
No, while the `ClearDefultCredentials` is set you not be able to connect to the network, because the SSID and password on the `wifi.config.yaml` were ignored.

Reply:
was this a feature request? Im having trouble seeing the value of this. If I want to remove my last credentials, i can just set 

```
Ssid: YourSSID
Password: SSIDPassword
```

and it'll replace my old creds, no? but well, if its optional, users can use this if they want.